src/components/label.ts

```diff
@@ -70,6 +72,16 @@ export class LabelComponent extends paper.Group {
     return this._background;
   }
 
+  /**
+   * Sets the font weight for the label. Updates background width to fit font change and checks for clipping.
+   * @param weight String or number value for font weight.
+   */
+  setFontWeight(weight: string | number) {
+    this._label.fontWeight = weight;
+    this.clip();
+    this._background.bounds.width = this._label.bounds.width + (LABEL_HORIZONTAL_PADDING * 2);
+  }
```

Comment:
would it be better to make this more generic and pass in a configuration object? ex. setFontStyle({ weight: 'bold' })